planning/behavior_path_planner/src/scene_module/lane_change/lane_change_module.cpp

```diff
@@ -73,7 +80,7 @@ void LaneChangeModule::onEntry()
 void LaneChangeModule::onExit()
 {
   resetParameters();
-  current_state_ = BT::NodeStatus::IDLE;
+  current_state_ = BT::NodeStatus::SUCCESS;
```

Comment:
https://github.com/BehaviorTree/BehaviorTree.CPP/blob/master/include/behaviortree_cpp/basic_types.h#L35

> /// IMPORTANT: Your custom nodes should NEVER return IDLE.

Reply:
I am not familiar with it much, but it may be better to change to SUCCESS also for other modules with another PR.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2391